src/Application/Digital/Commands/UploadDigitalFile.cs

```diff
@@ -89,6 +90,8 @@ public async Task<EntryDto> Handle(Command request, CancellationToken cancellati
             }
             else
             {
+                
+                entryEntity.Size = request.FileData!.Length;
```

Comment:
why don't move this down to centralize updating an entity?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/315/commits/80c9fb7b5afe06e2373c313e26bf54dd9fdf7406